services/siren/src/Lib.hs

```diff
@@ -136,7 +136,7 @@ copyScenario conn token userId authRole scId = do
 
 
 mkAuthRole :: Maybe AuthRole -> Maybe (Oid, ByteString, Format)
-mkAuthRole mauth = (\ar -> (oidTEXT, BSC.pack $ show ar, Text)) <$> mauth
+mkAuthRole mauth = (\ar -> Just (oidTEXT, BSC.pack $ show ar, Text)) $ fromMaybe Local mauth
```

Comment:
I decided against that because then you have to do the `fromMaybe` in every function because you still get a `Maybe AuthRole` as input.